### PR TITLE
Fix some issues with the launch parameters in the SUI

### PR DESCRIPTION
**Pull request by PankajBhojwani**
Addressing post-hoc comments on the launch parameters expander in the SUI (added in #13605)

- Use more contextually appropriate strings (`Centered` instead of `On` / `Off`)
- Don't emit an extra `NotifyChanges`

References #13605

### CODE REVIEW

File: src/cascadia/TerminalSettingsEditor/LaunchViewModel.cpp [hunk 1]
```diff
@@ -43,20 +43,23 @@ namespace winrt::Microsoft::Terminal::Settings::Editor::implementation
...
         else
         {
             const std::wstring xPosString = isnan(InitialPosX()) ? RS_(L"Globals_LaunchModeDefault/Content").c_str() : std::to_wstring(gsl::narrow_cast<int>(InitialPosX()));
             const std::wstring yPosString = isnan(InitialPosY()) ? RS_(L"Globals_LaunchModeDefault/Content").c_str() : std::to_wstring(gsl::narrow_cast<int>(InitialPosY()));
-            result = fmt::format(L"{}, ({},{}), {}", launchModeString, xPosString, yPosString, centerOnLaunchString);
+            result = fmt::format(L"{}, ({},{})", launchModeString, xPosString, yPosString);
         }
+
+        // Append the CenterOnLaunch part
+        result = CenterOnLaunch() ? fmt::format(L"{}, {}", result, RS_(L"Globals_CenterOnLaunchCentered")).c_str() : result;
```
Comment (DHowett) [inline src/cascadia/TerminalSettingsEditor/LaunchViewModel.cpp:62]: In general, if you find yourself constructing an hstring from a `c_str`, don't :smile:

(use `winrt::hstring{ fmt::format(...) }` if possible.)

File: src/cascadia/TerminalSettingsEditor/Resources/en-US/Resources.resw [hunk 2]
```diff
@@ -478,13 +478,9 @@
...
-  <data name="Globals_CenterOnLaunchOn" xml:space="preserve">
-    <value>On</value>
-    <comment>The "On" value for the center on launch setting.</comment>
-  </data>
-  <data name="Globals_CenterOnLaunchOff" xml:space="preserve">
-    <value>Off</value>
-    <comment>The "Off" value for the center on launch setting.</comment>
+  <data name="Globals_CenterOnLaunchCentered" xml:space="preserve">
+    <value>Centered</value>
+    <comment>What the "On" value for the center on launch setting represents.</comment>
```
Comment (DHowett) [inline src/cascadia/TerminalSettingsEditor/Resources/en-US/Resources.resw:483]: I might suggest, `Shorthand, explanatory text displayed when the user has enabled the feature indicated by `Globals_CenterOnLaunch.Text`

Comment (PankajBhojwani) [reply src/cascadia/TerminalSettingsEditor/Resources/en-US/Resources.resw]: Updated!

**Review by DHowett — APPROVED** [review]

**Review by DHowett — APPROVED** [review]

**Comment by DHowett** [comment]
@msftbot merge this in 10 minutes

File: src/cascadia/TerminalSettingsEditor/LaunchViewModel.cpp [hunk 3]
```diff
@@ -43,20 +43,23 @@ namespace winrt::Microsoft::Terminal::Settings::Editor::implementation
...
         else
         {
             const std::wstring xPosString = isnan(InitialPosX()) ? RS_(L"Globals_LaunchModeDefault/Content").c_str() : std::to_wstring(gsl::narrow_cast<int>(InitialPosX()));
             const std::wstring yPosString = isnan(InitialPosY()) ? RS_(L"Globals_LaunchModeDefault/Content").c_str() : std::to_wstring(gsl::narrow_cast<int>(InitialPosY()));
-            result = fmt::format(L"{}, ({},{}), {}", launchModeString, xPosString, yPosString, centerOnLaunchString);
+            result = fmt::format(L"{}, ({},{})", launchModeString, xPosString, yPosString);
         }
+
+        // Append the CenterOnLaunch part
+        result = CenterOnLaunch() ? winrt::hstring{ fmt::format(L"{}, {}", result, RS_(L"Globals_CenterOnLaunchCentered")) } : result;
```
Comment (lhecker) [inline src/cascadia/TerminalSettingsEditor/LaunchViewModel.cpp:62]: BTW I feel like this would be more readable if this was a regular if condition.

**Review by lhecker — APPROVED** [review]

**Pull request merged**